br/pkg/task/restore.go

```diff
@@ -27,7 +27,7 @@ const (
 	// FlagBatchFlushInterval controls after how long the restore batch would be auto sended.
 	FlagBatchFlushInterval = "batch-flush-interval"
 
-	defaultRestoreConcurrency = 128
+	defaultRestoreConcurrency = 512
```

Comment:
`512` is big. Default value would be better to be conservative. To avoid some new users crash their cluster.

Reply:
From the testing experience, `512` concurrency is much faster than `128` in restore process. If we keep the default `128` process, i think we should also keep `concurreny` as a visible parameter which is hidden now.https://github.com/tikv/migration/blob/d8c0fb1ba6f5d96dd43d8806c3082e152ef77696/br/pkg/task/common.go#L94

Reply:
I would like optimize the logic in next pr.